doc/rose-rug-advanced-tutorials-command-key.html

```diff
@@ -196,8 +190,8 @@ <h3 class="alwayshidden">Adding alternate commands (3)</h3>
       <p>Save your changes and run the suite. If you inspect the output from
       the breadmaker task you should see the line "dough for later".</p>
 
-      <p>Edit the command scripting for the <samp>[[breadmaker]]</samp> task to
-      change the command key to <samp>'timed_bread'</samp>. Run the suite and
+      <p>Edit the <code>script</code> for the <samp>[[breadmaker]]</samp> task
+      to change the command key to <samp>'timed_bread'</samp>. Run the suite and
       confirm the timed_bread command has been run.</p>
```

Comment:
As you're in here anyway, can you put samp tags round the secont timed_bread reference here.